lib/TPP/TileConsumerAndFuseProducers.cpp

```diff
@@ -25,108 +27,92 @@ using namespace mlir;
 
 namespace {
 
-// Check if loopRange is statically known.
-static bool isStaticRange(Range loopRange) {
-  Optional<int64_t> offsetAsInt = getConstantIntValue(loopRange.offset);
-  if (!offsetAsInt)
-    return false;
-  Optional<int64_t> sizeAsInt = getConstantIntValue(loopRange.size);
-  if (!sizeAsInt)
-    return false;
-  Optional<int64_t> strideAsInt = getConstantIntValue(loopRange.stride);
-  return static_cast<bool>(strideAsInt);
+// TODO: improve checks here.
```

Comment:
Be more specific on the kind of checks that are needed, so someone else can do that later, too.